soh/include/z64.h

```diff
@@ -30,7 +30,7 @@
 #include "ichain.h"
 #include "regs.h"
 
-#if defined(_WIN64) || defined(__x86_64__) || defined(__arm64__)
+#if defined(_WIN64) || defined(__x86_64__) || defined(__arm64__) || defined(__aarch64__) 
```

Comment:
this is just checking for bitness? 

```
__LP64__
```
should be defined on 64 bit platforms, might be a bit easier to see the intention.

Reply:
ooh, didn't know this one!